communication/best-practices.md

```diff
@@ -0,0 +1,82 @@
+# Mailing list and Github usage best practice                                                       
+                                                                                
+The Kubernetes Mailing list or Google Groups functions as the primary means of  
+asynchronous communication for the project's                                    
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your email account to attain a
+good signal-to-noise ratio with regards to the mailing list messages and Github
+notifications. All the steps below are basically for Gmail users.
```

Comment:
...but the same ideas should hold for other mail systems

Reply:
yes, but I am not sure that the same steps applies for other mail systems?